lib/Markdown.js

```diff
@@ -32,7 +32,7 @@ class Markdown extends Requestable {
     * @return {Promise} - the promise for the http request
     */
    render(options, cb) {
-      return this._request('POST', '/markdown', options, cb);
+      return this._request('POST', '/markdown', options, cb, true);
```

Comment:
I'm trying to figure out why sending the request as raw rather than JSON would work in the browser. Give me a bit to sort this out.